### ISSUE TITLE
Not getting ouput when I am not using DefaultPredictor

### ISSUE THREAD

**Issue by shreyjasuja** [facebookresearch/detectron2#2485]
I want to get mask pixels probability, so I didn't go by the traditional way of using Default Predictor, I used the snippet given below to get output. 

img = cv2.imread("/content/2.jpg")
img = np.transpose(img,(2,0,1))
img_tensor = torch.from_numpy(img)

cfg = get_cfg()

# add project-specific config (e.g., TensorMask) here if you're not running a model in detectron2's core library
cfg.merge_from_file(model_zoo.get_config_file("COCO-InstanceSegmentation/mask_rcnn_X_101_32x8d_FPN_3x.yaml"))
cfg.MODEL.ROI_HEADS.SCORE_THRESH_TEST = 0.7 # set threshold for this model
# Find a model from detectron2's model zoo. You can use the https://dl.fbaipublicfiles... url as well
model=detectron2.modeling.build_model(cfg)
DetectionCheckpointer(model).load("/content/model_final_2d9806.pkl")
model.train(False) 
inputs = [{"image":img_tensor}]
outputs = model(inputs)
#outputs = predictor(I'm)

The code neither throws any error nor gives any output :( 
I even tried using model.inference(inputs, do_postprocess=False)... that too fails to produce the output
